README.md

```diff
@@ -42,6 +42,27 @@
 
 <br />
 
+## 🖱️ One command Boyka project setup
+
+First you need to install [Boyka command line assistant](https://github.com/BoykaFramework/boyka-cli) by running the following command:
+
+```shell
+npm i -g @boykaframework/boyka-cli
+```
+
+Once it is installed, run the following command to setup Boyka project:
+
+```shell
+boyka init [project-name]
+```
+
+When you execute this command, you will be prompted by a set of questions which you must answer to setup the Boyka project with Maven and Java 17.
```

Comment:
When this command is executed, multiple set of questions will be prompted that must be answered to setup the project. By default the Boyka project uses Java 17 as programming language and Maven as build tool.